### DISCUSSION TITLE
getMonth returning wrong value with date generated by startOfYesterday or endOfYesterday

### DISCUSSION THREAD

**Discussion by jtshub** [js-fns/date-fns#2590] [Q&A] [answered]
I am calling startOfYesterday followed by getMonth and which gives the previous month.  For example today it 8/12

```
const prevDay = startOfYesterday();
// correctly returns 2021-08-11T04:00:00.000Z

const month = getMonth(prevDay);
// month = 7
```

If I I just create a date on my own it works
```
const prevDay = new Date(2021, 8, 11);
const month = getMonth(prevDay);
// month = 8
```

I assume this is bug?

**Comment by fturmel** [comment, marked as answer]
No bugs, months are 0-based indexes in JavaScript. The index of the month of August is 7 and not 8.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

```js
console.log(new Date(2021, 7, 11))
// Wed Aug 11 2021 00:00:00 GMT-0400 (Eastern Daylight Time)

console.log(new Date(2021, 8, 11))
// Sat Sep 11 2021 00:00:00 GMT-0400 (Eastern Daylight Time)
```
